denys/Sunshine/S03.01-Exercise-RecyclerView/app/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
+
+    defaultConfig {
+        applicationId "com.example.android.sunshine"
+        minSdkVersion 10
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(include: ['*.jar'], dir: 'libs')
+    compile 'com.android.support:appcompat-v7:25.1.0'
+
+    // TODO (1) Add RecyclerView dependency
+    compile 'com.android.support:recyclerview-v7:25.1.0'
+
+    // Instrumentation dependencies use androidTestCompile
+    // (as opposed to testCompile for local unit tests run in the JVM)
+    androidTestCompile 'junit:junit:4.12'
+    androidTestCompile 'com.android.support:support-annotations:25.1.0'
```

Comment:
Consider adding the support version (here 25.1.0) as a variable in the top level gradle build file, since you use the same version several places it will be easier to change when there is an update. Made a comment at the top level build file as well.